bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WpToBpTop/WpToBpT_conf/Zt_proc_card.dat

```diff
@@ -0,0 +1,5 @@
+import model VLQVP_UFO
+generate p p > wp+ bp~ > z t b~
```

Comment:
Does "wp+ decay to bp top" and "bp to z and b"?
The syntax is not so safe in some sense so it would be my naive suggestion to change like below :
p p > t bp~, bp~ > z b~
Of course this case you are assuming onshell bp production so it only works when you are doing some bump search for m(bp)

Reply:
MassesWp = [2000, 3000, 5000]
MassesBp = [25,80,170,400]
MassesT = [25,80,170,400]

But looking at your mass settings m(Wp) >> m(Bp) so I think the syntax suggested would be OK